addons/website/data/website_data.xml

```diff
@@ -601,6 +601,24 @@
             <field name="type">url</field>
             <field name="url">/website/static/src/img/snippets_demo/s_text_image.jpg</field>
         </record>
+        <record id="website.s_three_columns_image_default_1" model="ir.attachment">
```

Comment:
@Cocographique you named them image_default here but used default_image in the template so this does not work :/ Anyways, I fixed it.